pkg/fixer/fixer.go

```diff
@@ -0,0 +1,117 @@
+package fixer
+
+import (
+	"fmt"
+	"io"
+
+	"github.com/open-policy-agent/opa/ast"
+	"github.com/open-policy-agent/opa/format"
+
+	"github.com/styrainc/regal/pkg/fixer/fixes"
+	"github.com/styrainc/regal/pkg/report"
+)
+
+func Fix(rep *report.Report, readers map[string]io.Reader, _ fixes.Options) (map[string][]byte, error) {
```

Comment:
the idea with the readers here was that it made the impl not dependent on files. However, I'm not sure it's all that clean because of it.